msteams-platform/toolkit/provision.md

```diff
@@ -1,616 +1,436 @@
+You can follow below steps to add environment variables to the .env files to use an Azure AD app created for your Teams app. If you don't have an Azure AD app for your bot yet or you already have one but don't know where to find the correct values, see [how to use existing Azure AD app in TeamsFx project](https://github.com/devdiv-azure-service-dmitryr/teamsfx-docs/blob/main/V5-doc-update/use-existing-aad-app.md).
 
-> [!NOTE]
-> Ensure that not to share the same Azure AD app in multiple environments. If you don't have permission to update the Azure AD app, you get a warning with instructions to manually update the Azure AD app. Follow these instructions to update your Azure AD app after provision.
+1. Open `teamsapp.yml` and find the `botAadApp/create` action.
 
-#### Use an existing Azure AD app for your bot
+1. Find the environment variable names that store information for Azure AD app in the `writeToEnvironmentFile` property. Below is the default `writeToEnvironmentFile` definition if you create projects using Teams Toolkit:
 
-You can add the following configuration snippet to `.fx\configs\config.{env}.json` file to use the Azure AD app created for your bot:
+ ```typescript
+ writeToEnvironmentFile:
+   botId: BOT_ID
+   botPassword: SECRET_BOT_PASSWORD
+ ```
 
-```json
-"bot": {
-    "appId": "<your Azure AD app client id>",
-    "appPassword": "{{$env.ENV_NAME_THAT_STORES_YOUR_SECRET}}"
-}
-```
+1. Add values for each environment variable from step 2.
 
-After adding the snippet, add your client secret to the related environment variable for Teams Toolkit to resolve the actual client secret during provision.
+    1. Add below environment variable and its value to `env\.env.{env}` file.
 
-#### Skip adding user for SQL database
+    ```typescript
+    BOT_ID=<value of Azure AD application's client id (application id)> # example: 00000000-0000-0000-0000-000000000000
+    ```
 
-If you get an insufficient permission error when Teams Toolkit tries to add user to SQL database, add the following configuration snippet to `.fx\configs\config.{env}.json` file to skip adding SQL database user:
+    1. Add below environment variable and its value to `env\.env.{env}.user` file.
 
-```json
-"skipAddingSqlUser": true
-```
+    ```typescript
+    SECRET_BOT_PASSWORD=<value of Azure AD application's client secret>
+    ```
 
-::: zone-end
+>[!NOTE]
+>
+> * Remember to update the environment variable names in the examples if you use different names in `writeToEnvironmentFile`.
+>
+> * If you don't use `botAadApp/create` action to create Azure AD application, you can add necessary environment variables with your preferred name without following above steps.
+>
+> * Ensure not to share the same Azure AD app in multiple environments. If you don't have permission to update the Azure AD app, you get a warning with instructions about how to manually update the Azure AD app. Follow the instructions to update your Azure AD app after provision.
```

Comment:
Only keep `Ensure not to share the same Azure AD app in multiple environments.` and remove remaining part.

Reply:
Updated.